cli/tests/integrations/test_dcos.py

```diff
@@ -62,6 +62,12 @@ def test_help():
 
     DCOS_DEBUG                  If set then enable further debug messages which
                                 are sent to stdout.
+
+    DCOS_SSL_VERIFY             If set specifies whether to verify SSL certs
+                                for HTTPS.
+
+    DCOS_CERT                   This environment variable points to the
+                                location of the SSL cert .pem file
```

Comment:
Let's document this better. Please state that the user can control the value of these environment variables by using `dcos config set ...`.